opensearch-operator/pkg/builders/cluster.go

```diff
@@ -193,6 +194,12 @@ func NewSTSForNodePool(
 
 	var mainCommand []string
 	com := "./bin/opensearch-plugin install --batch"
+	if cr.Spec.General.Monitoring {
+		clusterVersion := cr.Spec.General.Version
+		url := "https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/" + clusterVersion + ".0/prometheus-exporter" + clusterVersion + ".0.zip"
```

Comment:
Hey @idanl21 will this work for operator offline setup?, Like when operator installed in a restrictive environment with limited internet URL access, can it then pull this plugin?

Reply:
Probably it will not work, maybe we can add a guide, on how to download it and store it in a place where you can pull it. have another idea?
by the way, I think that problem will be also a problem with official OpenSearch plugins.